addons/viewports/CfgVehicles.hpp

```diff
@@ -0,0 +1,169 @@
+class CfgVehicles {
+    class B_MBT_01_base_F;
+    class B_MBT_01_cannon_F: B_MBT_01_base_F { // Merkel
+        class ADDON {
+            class SLD_backLeftUpper {
+                type = "screen";
+                camLocation[] = {0,0,0.05};
+                maxDistance = 5;
+                camAttach[] = {0,0};
+                screenLocation[] = {-0.925,-3.43459,-1.07};
+                roles[]={"cargo"};
+            };
+        };
+    };
+    class B_MBT_01_TUSK_F: B_MBT_01_cannon_F { // Merkel TUSK
+        class ADDON {
+            class SLD_backLeftUpper {
+                type = "screen";
+                camLocation[] = {0,0,0.05};
+                maxDistance = 5;
+                camAttach[] = {0,0};
+                screenLocation[] = {-0.925,-4.65511,-1.07};
+                roles[]={"cargo"};
+            };
+        };
+    };
+
+
+
+    class Wheeled_APC_F;
+    class Tank_F;
+
+     // 80 and 80A p3ds both use same modelPos
+    #if __has_include("\CUP\WheeledVehicles\CUP_WheeledVehicles_BTR80\CUP_BTR_80.p3d")
```

Comment:
Does HEMTT support leaving a config.cpp unbinarized?
Would be cool to add a 
#pragma NO_BINARIZE
or something like that, I can maybe add that to CfgConvert/AddonBuilder in 2.08

Reply:
no, it does not and it does not even have support for #if nor __has_include